option optimize_for = LITE_RUNTIME;

message EquipStrengthenRequest
{
 required uint32 posZhengXing = 1;
 required uint32 pos = 2;
 required uint32 times = 3;
}
